ondemand.osc.edu/central_config.yaml

```diff
@@ -0,0 +1,261 @@
+OOD_DASHBOARD_DEV_DOCS_URL: null
+## OOD_DASHBOARD_DOCS_URL
+# Purpose: An external link to documentation for OnDemand users
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_DOCS_URL: https://www.osc.edu/ondemand
+## OOD_DASHBOARD_PASSWD_URL
+# Purpose: An external link to your site's password changing service
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_PASSWD_URL: https://my.osc.edu
+## OOD_DASHBOARD_SUPPORT_URL
+# Purpose: An external link to your site's user support
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_SUPPORT_URL: https://www.osc.edu/contact/supercomputing_support
+## OOD_DASHBOARD_HEADER_IMG_LOGO
+# Purpose: ???
+# Type: String | null
+# Default:
+OOD_DASHBOARD_HEADER_IMG_LOGO: null
+## OOD_DASHBOARD_LOGO
+# Purpose: A link to an image to use in the Dashboard's header
+# Replaces: DISABLE_DASHBOARD_LOGO
+# Example: /public/logo.png
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_LOGO: /public/logo.png
+## OOD_DASHBOARD_SUPPORT_EMAIL
+# Purpose: The email address for your site's user support staff
+# Type: String | null
+# Default:
+OOD_DASHBOARD_SUPPORT_EMAIL: oschelp@osc.edu
+## OOD_DASHBOARD_TITLE
+# Purpose: The title for your OnDemand instance; displayed on the Dashboard
+# Type: String | null
+# Default: Open OnDemand
+OOD_DASHBOARD_TITLE: OSC OnDemand
+## OOD_DATAROOT
+# TODO: revisit this
+# Purpose: ???
+# Type: String | null
+# Default:
+OOD_DATAROOT: null
+## OOD_DEV_APPS_ROOT
+# Purpose: Contains the path to the user-specific development directory
+# TODO: What sets this? Do we ever want to override it? Is it always user-specific?
+# Type: String | null
+# Default: /dev/null
+OOD_DEV_APPS_ROOT: null
+## OOD_DEV_SSH_HOST
+# Purpose: A web node, serving OnDemand, where developers can SSH to perform development work.
+# Type: String | null
+# Default:
+OOD_DEV_SSH_HOST: ondemand-test.osc.edu
+## OOD_FILE_EDITOR
+# Purpose: A web-servable path to an alternative file editor application
+# Type: String | null
+# Default: /pun/sys/file-editor/edit
+OOD_FILE_EDITOR: null
+## OOD_LOAD_EXTERNAL_BC_CONFIG
+# Purpose: Flags whether to load external configuration for Batch Connect apps; impacts sub apps
+# TODO: Help me understand sub apps
+# Type: Boolean | null
+# Default: true
+OOD_LOAD_EXTERNAL_BC_CONFIG: false
+## OOD_LOAD_EXTERNAL_CONFIG
+# Purpose: Flags whether to load external configuration; includes initializers, and custom views
+# Type: Boolean | null
+# Default: true
+OOD_LOAD_EXTERNAL_CONFIG: false
+## OOD_LOCALE
+# Purpose: Sets the locale/translation to use
+# Type: String | null
+# Default: en
+OOD_LOCALE: null
+## OOD_LOCALES_ROOT
+# Purpose: File system path where localizations are stored
+# Type: String | null
+# Default: /etc/ood/config/locales
+OOD_LOCALES_ROOT: null
+## OOD_NAVBAR_TYPE
+# Purpose: The Boostrap navbar specialization
+# Type: String | null
+# Default: inverse
+OOD_NAVBAR_TYPE: null
+## OOD_PORTAL
+# Purpose: ???
+# Type: String | null
+# Default:
+OOD_PORTAL: null
```

Comment:
This is set in nginx_stage config and should not be set here

Reply:
`OOD_PORTAL` that is

Reply:
Removed.